harness/determined/common/api/request.py

```diff
@@ -44,7 +45,12 @@ def maybe_upgrade_ws_scheme(master_address: str) -> str:
 
 
 def make_interactive_task_url(
-    task_id: str, service_address: str, description: str, resource_pool: str, task_type: str
+    task_id: str,
+    service_address: str,
+    description: str,
+    resource_pool: str,
+    task_type: str,
+    currentSlotsExceeded: bool,
```

Comment:
Out of curiosity, why is this value passed through the client, and not through the master?

Reply:
could you elaborate on why you think this value should be passed through the master?